vendor/index.yaml

```diff
@@ -1,7 +1,15 @@
 vendors:
+  - id: example
+    name: Example vendor with XYZ devices
+    website: https://www.example.com
+    logo: example/logo.svg
   - id: bosch
     name: Bosch Connected Devices and Solutions GmbH
     pen: 42
     oui: FCD6BD
     website: https://www.bosch-connectivity.com
     logo: bosch/logo.svg
+  - id: decentlab
+    name: Decentlab GmbH
+    website: https://www.decentlab.com/
+    logo: decentlab/decentlab-logo.svg
```

Comment:
I'm sure they have an OUI, and probably also a PEN

Reply:
Didn't find Decentlab in the files:
- http://standards-oui.ieee.org/oui.txt
- https://www.iana.org/assignments/enterprise-numbers/enterprise-numbers